docs/topic/troubleshooting.md

```diff
@@ -49,3 +52,28 @@ prometheus:
 The [default for this value is 2GB](https://github.com/2i2c-org/infrastructure/tree/HEAD/helm-charts/support/values.yaml#L38).
 Try increasing this value in small increments and deploying using `python deployer deploy-support {{CLUSTER_NAME}}` until the server successfully starts up.
 Then make a Pull Request with your updated config.
+
+(troubleshooting/validation-webhook)=
+## Failure calling validation webhook for `support` components
+
+After addressing [](troubleshooting/prometheus-oom), you may see the following error when redeploying the support chart:
```

Comment:
I'm actually not sure I've got referencing right here - I can never remember sphinx vs jupyter book vs something else syntax!